src/main/scala/mesosphere/marathon/api/TaskKiller.scala

```diff
@@ -55,14 +55,14 @@ class TaskKiller @Inject() (
   }
 
   private[this] def expunge(tasks: Iterable[Instance])(implicit ec: ExecutionContext): Future[Unit] = {
-    // Note: We process all tasks sequentially.
+    // Note: We process all instances sequentially.
 
-    tasks.foldLeft(Future.successful(())) { (resultSoFar, nextTask) =>
+    tasks.foldLeft(Future.successful(())) { (resultSoFar, nextInstance) =>
```

Comment:
Is there a reason we need to do this in serial? If so, maybe document way? Also, perhaps a good time to switch over to Future[Done]?